jax/interpreters/ad.py

```diff
@@ -296,13 +300,17 @@ def process_custom_jvp_call(self, _, __, f_jvp, tracers):
     primals_in, tangents_in = unzip2((t.primal, t.tangent) for t in tracers)
     primals_in = map(core.full_lower, primals_in)
     tangents_in = map(instantiate_zeros, tangents_in)
+    tangents_in = [aval.to_zeros() if type(aval) is core.Float0Array
+                   else aval for aval in tangents_in]
     outs = f_jvp.call_wrapped(*it.chain(primals_in, tangents_in))
     primals_out, tangents_out = split_list(outs, [len(outs) // 2])
     return map(partial(JVPTracer, self), primals_out, tangents_out)
 
   def process_custom_vjp_call(self, _, __, fwd, bwd, tracers, *, out_trees):
     primals_in, tangents_in = unzip2((t.primal, t.tangent) for t in tracers)
     tangents_in = map(instantiate_zeros, tangents_in)
+    tangents_in = [aval.to_zeros() if type(aval) is core.Float0Array
```

Comment:
Similar to the above:

```suggestion
    tangents_in = [t.to_zeros() if type(t) is core.Float0Array else t for t in tangents_in]
```